frontend_tests/node_tests/channel.js

```diff
@@ -277,7 +266,9 @@ function test_with_mock_ajax(test_params) {
                 },
             });
 
-            assert(logged);
+            assert.equal(blueslip.get_test_logs('log').length, 1);
+            assert.equal(blueslip.get_test_logs('log')[0], 'Retrying idempotent[object Object]');
```

Comment:
Is there any reason why you use `set_test_data` and just `get_test_logs` in some places?

Reply:
Hey, sorry for the delayed response. The module is set to raise an error when an unexpected `error, warning or fatal` call is executed. For `log, debug and info`, it silently just logs whatever argument it has been called with. Thus, we need to set expected errors for the calls to errors etc, and can just check the test logs for the logs etc.

Reply:
Ah, got it.